foundry/applications/formApplications/baseEntitySheets/journalSheet.d.ts

```diff
@@ -5,7 +5,11 @@ declare class JournalSheet<
   D extends object = BaseEntitySheet.Data<JournalEntry>,
   O extends JournalEntry = D extends BaseEntitySheet.Data<infer T> ? T : JournalEntry
 > extends BaseEntitySheet<D, O> {
-  constructor(object: O, options?: JournalSheet.Options);
+  /**
+   * @param entity - The JournalEntry instance which is being edited
```

Comment:
Just one more small nitpick regarding style: Please align the `-` of the params.